src/SignatureSectionProcessor.h

```diff
@@ -431,6 +433,10 @@ namespace scpl {
                 }
             }
 
+            if (i == subject.length() && clearAtEnd) {
```

Comment:
Just a minor thing: `clearAtEnd && i == subject.length()` would be more effective as the length of the subject does have to be computed if `clearAtEnd` is truthy